cmd/podman/images/pull.go

```diff
@@ -154,17 +155,19 @@ func imagePull(cmd *cobra.Command, args []string) error {
 	}
 	// Let's do all the remaining Yoga in the API to prevent us from
 	// scattering logic across (too) many parts of the code.
-	pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), args[0], pullOptions.ImagePullOptions)
-	if err != nil {
-		return err
-	}
-
-	if len(pullReport.Images) > 1 {
-		fmt.Println("Pulled Images:")
+	var errs utils.OutputErrors
+	for _, arg := range args {
+		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+		for _, img := range pullReport.Images {
+			fmt.Println(img)
+		}
 	}
-	for _, img := range pullReport.Images {
-		fmt.Println(img)
+	if len(errs) > 0 {
```

Comment:
No need for this check, if errs.PrintErrors will return nil if there are no errors.
LGTM